core/domain/subscription_services.py

```diff
@@ -206,6 +236,10 @@ def record_user_has_seen_notifications(user_id, last_seen_msecs):
     """Updates the last_checked time for this user (which represents the time
     the user last saw the notifications in the dashboard page or the
     notifications dropdown).
+    Args:
+        user_id: str. User IDself.
```

Comment:
This description looks like it needs fixing.

Reply:
Done!